### PR TITLE
docs(story): asset documentpanel in new format

### PR DESCRIPTION
**Pull request by amevck**
For getting comment

### CODE REVIEW

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-524.gearboxjs.preview.cogniteapp.com

File: src/components/AssetDocumentsPanel/stories/basic.story.mdx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+This component loads list of documents related to the asset and presents it in the list of categories.
+Each item in the list can be expanded showing list of files.
+The component is used as one of the panes in `AssetMeta` component.
+
+**NOTE:** The component should have `ClientSDKProvider` as a parent component in react component tree.
+
+## Basic usage:
+
+Don't forget to import:
```
Comment (polomani) [inline src/components/AssetDocumentsPanel/stories/basic.story.mdx]: "Don't forget" is something that you say after you gave some instructions before
but doesn't really sound good on the beginning.
can be replaced with just `Imports:`

File: src/components/AssetDocumentsPanel/stories/basic.story.mdx [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+This type can be imported from `@cognite/sdk`:
+
+```jsx
+import { FileRequestFilter } from `@cognite/sdk`;
+```
+
+#### OnDocumentClick
+
+Definition:
```
Comment (polomani) [inline src/components/AssetDocumentsPanel/stories/basic.story.mdx]:
```suggestion
```
can be just removed, seems like no extra explanation is needed

File: src/components/AssetDocumentsPanel/stories/basic.story.mdx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+#### CollapseProps
+This type can be imported from `antd`:
+
+```jsx
+import { CollapseProps } from 'antd/lib/collapse';
+```
+
+#### DocumentRenderer
+
+Definition:
```
Comment (polomani) [inline src/components/AssetDocumentsPanel/stories/basic.story.mdx]:
```suggestion
```
same as above

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-524.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-524.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-524.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-524.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-524.gearboxjs.preview.cogniteapp.com

File: src/components/AssetDocumentsPanel/stories/helper.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+import { DOCUMENTS } from '../../../mocks';
+import { MockCogniteClient } from '../../../utils/mockSdk';
+import { ClientSDKProvider } from '../../ClientSDKProvider';
+class CogniteClient extends MockCogniteClient {
+  files: any = {
+    list: () => ({
+      autoPagingToArray: () => {
+        return new Promise(resolve => {
+          setTimeout(() => {
+            resolve(DOCUMENTS);
```
Comment (polomani) [inline src/components/AssetDocumentsPanel/stories/helper.tsx]: you can use `import { sleep } from './common';` instead of this

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-524.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-524.gearboxjs.preview.cogniteapp.com

**Pull request merged**